config/test_cases/test_pull_request_openshift_ansible_tox.yml

```diff
@@ -18,6 +18,20 @@ actions:
     repository: "openshift-ansible"
     script: |-
       tox
+  - type: "script"
```

Comment:
I don't think it's the long-term location we want this, but the tox job is pretty short and @stevekuznetsov thought it would be easiest to just start out by extending it rather than create a new job. As integration tests get unwieldy, they could be moved to a new job(s).

Note that there's a build run for every test... right now builds are pretty quick, but I could see them getting a lot longer in the future. We didn't do the builds separately at this time as there's no really good place to push them to (private registry hopefully coming "soon").